src/uct/api/uct_def.h

```diff
@@ -360,7 +360,16 @@ typedef struct uct_cm_ep_client_connect_args {
     const uct_cm_remote_data_t *remote_data;
 
     /**
-     * Indicates the server's @ref ucs_status_t status.
+     * Indicates the server's @ref ucs_status_t status:
+     * UCS_OK                   - the server accepted the client's connection
+     *                            establishment request.
+     * UCS_ERR_REJECTED         - the server rejected the client's connection
+     *                            establishment request.
+     * UCS_ERR_CONNECTION_RESET - the server's connection was reset during
+     *                            the connection establishment to the client.
+     * Other                    - an internal (client's) local error on the
+     *                            sockaddr transport while establishing a
+     *                            connection to the server.
```

Comment:
Otherwise - indicates an internal connection establishment error on the local (client) side